### PR TITLE
MISUV-7358 Integration Tests Setup

### PR DESCRIPTION
**Pull request by danielmaltz1**

### CODE REVIEW

File: app/uk/gov/hmrc/incometaxsessiondata/services/SessionService.scala [hunk 1]
```diff
@@ -38,4 +38,8 @@ class SessionService @Inject()(
   def set(sessionData: SessionData): Future[Boolean] = {
     repository.set(sessionData)
   }
+
+  def deleteSession(sessionId: String): Future[Boolean] = {
```
Comment (avelx) [inline app/uk/gov/hmrc/incometaxsessiondata/services/SessionService.scala:42]: if we can apply same approach as in this discussion:

https://hmrcdigital.slack.com/archives/C050V2P86BC/p1712737921076559?thread_ts=1712737564.343879&cid=C050V2P86BC

Comment (danielmaltz1) [reply app/uk/gov/hmrc/incometaxsessiondata/services/SessionService.scala:42]: Hi Pavel, do you mean just change the return type to Future[Unit]? I use the same approach as Siddharth on the repository level already

Comment (danielmaltz1) [reply app/uk/gov/hmrc/incometaxsessiondata/services/SessionService.scala]: I have done this now

File: project/CodeCoverageSettings.scala [hunk 2]
```diff
@@ -16,7 +16,7 @@ object CodeCoverageSettings {
 
   val settings: Seq[Setting[_]] = Seq(
     ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
-    ScoverageKeys.coverageMinimumStmtTotal := 65,
+    ScoverageKeys.coverageMinimumStmtTotal := 90,
```
Comment (avelx) [inline project/CodeCoverageSettings.scala:19]: :smiley:

**Review by avelx — APPROVED** [review]

**Pull request merged**